package.json

```diff
@@ -114,7 +114,8 @@
         ],
         "testMatch": [
             "**/src/**/__tests__/**/*.[jt]s?(x)",
-            "**/src/**/?(*.)+(spec|test).[jt]s?(x)"
+            "**/src/**/?(*.)+(spec|test).[jt]s?(x)",
+            "**/test/**?(*.)+(spec|test).[jt]s?(x)"
```

Comment:
Do you mind if we stick with the `__tests__` pattern? It's a Jest standard, and the underscores make it easier to visually scan the folder structure when looking for files

Reply:
ohh, that's crazy, i didn't know about the `/test/`!